src/retrace/retrace_worker.py

```diff
@@ -473,20 +472,14 @@ def start_retrace(self, custom_arch=None):
         # create mock config file
         try:
             repopath = os.path.join(CONFIG["RepoDir"], releaseid)
-            linux_dist = distro.linux_distribution(full_distribution_name=False)
             with open(os.path.join(task.get_savedir(), RetraceTask.MOCK_DEFAULT_CFG), "w") as mockcfg:
                 mockcfg.write("config_opts['root'] = '%d'\n" % task.get_taskid())
                 mockcfg.write("config_opts['target_arch'] = '%s'\n" % arch)
-                mockcfg.write("config_opts['chroot_setup_cmd'] = '")
-                if linux_dist[0] == "fedora":
-                    mockcfg.write("--setopt=strict=0")
-                else:
-                    mockcfg.write("--skip-broken")
+                mockcfg.write("config_opts['chroot_setup_cmd'] = '--skip-broken")
                 mockcfg.write(" install %s abrt-addon-ccpp shadow-utils %s rpm'\n" % (" ".join(packages),
                                                                                       self.plugin.gdb_package))
-                mockcfg.write("config_opts['releasever'] = '%s'\n" % linux_dist[1])
-                if linux_dist[0] == "fedora":
-                    mockcfg.write("config_opts['package_manager'] = 'dnf'\n")
+                mockcfg.write("config_opts['releasever'] = '%s'\n" % version)
+                mockcfg.write("config_opts['package_manager'] = 'dnf'\n")
```

Comment:
Should I remove this line then?

Reply:
I’m thinking not, since you say that we’re missing dependencies for actual support. Would be nice if things just worked, but oh well.